README.md

```diff
@@ -1,88 +1,80 @@
-# MongoDB Hot Backup
+# mongo-hot-backup
+
+[![Circle CI](https://circleci.com/gh/Financial-Times/mongo-hot-backup/tree/master.png?style=shield)](https://circleci.com/gh/Financial-Times/mongo-hot-backup/tree/master)[![Go Report Card](https://goreportcard.com/badge/github.com/Financial-Times/mongo-hot-backup)](https://goreportcard.com/report/github.com/Financial-Times/mongo-hot-backup) [![Coverage Status](https://coveralls.io/repos/github/Financial-Times/mongo-hot-backup/badge.svg)](https://coveralls.io/github/Financial-Times/mongo-hot-backup)
 
 This tool can back up or restore MongoDB collections while DB is running to/from AWS S3.
 
-# Usage
-## Build/install
-### Go app
-```
-go get -u github.com/Financial-Times/mongodb-hot-backup
-```
-### Docker app
+You can deploy a docker container that will run backups on schedule (default at 10:30am every day). Or you can just run the container to make a single backup, or restore from a given point of time.
+
+For the schedule option, the state of backups is kept in a boltdb file at `/var/data/mongo-hot-backup/state.db`
+
+Health endpoint is available at `0.0.0.0:8080/__health` and will report healthy if there was a successful backup for each configured collection in the last 26 hours.
+
+An initial backup will be ran upon startup if there's no backup found in the last 26 hours. Can be disabled.
+
+## Installation
+
 ```
-docker build -t coco/mongodb-hot-backup   .
+go get -u github.com/Financial-Times/mongo-hot-backup
 ```
 
-## Run
-### Backup
+## Building
```

Comment:
You should also add the fact that you need to install `govendor` and run `govendor sync`.

Reply:
ok